packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -64,9 +64,9 @@ export const indexPage =
 
       // If there is any localData, display hyperlinks to each page that will be generated
       // from each data document.
-      if (localDataManifest.static.length + localDataManifest.entity.size) {
+      if (localDataManifest.static.size + localDataManifest.entity.size) {
```

Comment:
I think we should be using length over size: https://stackoverflow.com/questions/14202601/array-size-vs-array-length